zulip_bots/zulip_bots/bots/xkcd/xkcd.py

```diff
@@ -14,6 +16,17 @@ class XkcdHandler(object):
     commands.
     '''
 
+    META = {
+        'name': 'XKCD',
+        'description': 'Fetches comic strips from https://xkcd.com.',
+        'defaults': True,
+        'commands': OrderedDict([
+                ('latest', "Show the latest comic strip"),
+                ('random', "Show a random comic strip"),
+                ('<comic id>', "Show a comic strip with a specific 'comic id'"),
+        ])  # NOTE: help not listed here, so default command used
```

Comment:
In this bot, with this PR, help is now handled internally by the lib, so the default command is used instead as it stands. If 'help' was listed in `commands` then the bot wouldn't handle it and the help text would be as specified in the tuple.